app/plugin.go

```diff
@@ -457,6 +476,20 @@ func (ch *Channels) disablePlugin(id string) *model.AppError {
 	return nil
 }
 
+func (ch *Channels) notifyInstalledIntegrationsChanged() *model.AppError {
+	integrations, appErr := ch.getInstalledIntegrations()
+	if appErr != nil {
+		return appErr
+	}
+
+	message := model.NewWebSocketEvent(model.WebsocketEventInstalledIntegrationsChanged, "", "", "", nil)
+	message.Add("integrations", integrations)
+	message.GetBroadcast().ContainsSensitiveData = true
```

Comment:
Ditto comment about needing an (updated) enabled plugin count for regular users. I'd be ok modifying this to only return the count or to have two events, one that returns this and one that returns the count of enabled.